react/features/connection-indicator/components/web/ConnectionIndicator.js

```diff
@@ -309,7 +312,16 @@ class ConnectionIndicator extends AbstractConnectionIndicator<Props, State> {
     _renderIcon() {
         const colorClass = this._getConnectionColorClass();
 
-        if (this.props._connectionStatus === JitsiParticipantConnectionStatus.INACTIVE) {
+        let isConnectionStatusInactive = this.props._connectionStatus === JitsiParticipantConnectionStatus.INACTIVE;
+        let isConnectionStatusInterrupted
+            = this.props._connectionStatus === JitsiParticipantConnectionStatus.INTERRUPTED;
+
+        if (this.props._sourceNameSignalingEnabled) {
+            isConnectionStatusInactive = this.props._streamingStatus === JitsiTrackStreamingStatus.INACTIVE;
```

Comment:
It seemed a bit redundant to use `JitsiTrackStreamingStatus` which is exactly the same as `JitsiParticipantConnectionStatus` but I might be cleaner to not mix the two

Reply:
yes, it's better to not mix the two